website/docs/docs/quickstarts/dbt-core/codespace-qs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Quickstart for dbt Core using GitHub Codespaces"
+id: codespace
+description: "Quickly get set up to start running dbt commands in a codespace environment."
+sidebar_label: "GitHub Codespace quickstart"
+---
+
+In this quickstart guide, you’ll learn how to create a codespace and be able to execute the `dbt build` command from it in _less than 5 minutes_. 
+
+dbt Labs provides a [GitHub Codespace](https://docs.github.com/en/codespaces/overview) template that you (and anyone else) can reuse to create a complete dbt environment with a working and runnable project. When you create the codespace, the [dev container](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers) creates a fully functioning dbt environment, connects to a DuckDB database, and loads a year of fictional Jaffle Shop data into the database for you. Jaffle Shop is a fictional cafe selling food and beverages in several US cities.
+
+:::tip Check out our Jaffle Shop template 
+The [README](https://github.com/dbt-labs/jaffle-shop-template#readme) for the Jaffle Shop template also provides instructions on how to do this, along with animated GIFs. You can check those out, too.  
+
+:::
+
+## Prerequisites
+
+- To use the dbt CLI, it's important that you know some basics of the terminal. In particular, you should understand `cd`, `ls` , and `pwd` to navigate through the directory structure of your computer easily.
```

Comment:
Since this is for beginners:

```suggestion
- To use the dbt command line interface (CLI), it's important that you know some basics of the terminal. In particular, you should understand `cd`, `ls` , and `pwd` to navigate through the directory structure of your computer easily.
```